modules/stardate/tests/test_stardate.c

```diff
@@ -37,10 +37,13 @@ stardate_assert(const gchar *msg_str, const int precision, const gchar *expected
   LogMessage *logmsg = log_msg_new(msg_str, strlen(msg_str), NULL, &parse_options);
 
   char *template_command;
+  int ret_val;
   if (precision == -1)
-    asprintf(&template_command, "$(stardate $UNIXTIME)");
+    ret_val = asprintf(&template_command, "$(stardate $UNIXTIME)");
   else
-    asprintf(&template_command, "$(stardate --digits %d $UNIXTIME)", precision);
+    ret_val = asprintf(&template_command, "$(stardate --digits %d $UNIXTIME)", precision);
+
+  assert_false(ret_val == -1, "Memory allocation failed in asprintf.");
```

Comment:
A lot is happening in this patch. I suggest a reword at least, the fixing indentation as a summary is misleading. Maybe worth splitting this patch?

Reply:
This again was cosmetic and fixed, though github cannot detect the fix. I remove the comments so not to confuse others.